server/controllers/courses/addCourse.js

```diff
@@ -0,0 +1,29 @@
+const joi = require('joi');
+const { insertCourse, insertDates } = require('../../database/queries/addCourse');
+const { addCourseSchema } = require('../../helpers/validation-schema');
+
+exports.addCourse = (req, res) => {
+  const { error } = joi.validate({ ...req.body }, addCourseSchema);
+  if (error) {
+    res.send({ msg: error.details[0].message });
```

Comment:
send the status code 
```suggestion
    res.status(400).send({ msg: error.details[0].message });
```